zerver/lib/markdown/preprocessor_priorities.py

```diff
@@ -10,9 +10,12 @@
     "generate_return_values": 510,
     "generate_api_arguments": 505,
     "include": 500,
+    "include_wrapper": 500,
     "help_relative_links": 475,
     "setting": 450,
+    "html_block": 20,
     "fenced_code_block": 25,
+    "normalize_whitespace": 30,
```

Comment:
These should be commented out, I agree with @timabbott

Reply:
And maybe add a comment at the top that says that the ones that are commented out are built-in preprocessor priorities.

Reply:
@eeshangarg Thanks for the input! Just to be clear, the 3 new lines I added here are built-in preprocessor priorities and should be commented out? i'll add a new commit shortly.

Reply:
@kevjn Yes, the three built-in priorities should be commented out.